tools/config-generator/periodic_config.go

```diff
@@ -226,7 +257,35 @@ func generatePeriodic(title string, repoName string, periodicConfig yaml.MapSlic
 	}
 	addExtraEnvVarsToJob(extraEnvVars, &data.Base)
 	configureServiceAccountForJob(&data.Base)
+
+	// This is where the data actually gets written out
 	executeJobTemplate("periodic", jobTemplate, title, repoName, data.PeriodicJobName, false, data)
+
+	// If job is a continuous run, add a duplicate for pre-release testing of new prow-tests image
+	// It will run less often than source job
+	if isContinuousJob {
+		betaData := data.Clone()
+
+		// Change the name and image
+		betaData.PeriodicJobName += "-beta-prow-tests"
+		betaData.Base.Image = strings.ReplaceAll(betaData.Base.Image, ":stable", ":beta")
+
+		// Run 2 or 4 times a day
```

Comment:
The frequency of continuous job was computed based on timeout, if you don't like to change the timeout of these beta test jobs, at least there should be comment explaining the reason why not use generateCron function

Reply:
Sure, it's because their purpose is to test the beta prow-tests and has nothing to do with whatever the underlying job is. I will make it clearer.